datasets/hind_encorp/hind_encorp.py

```diff
@@ -0,0 +1,172 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""TODO: Add a description here."""
+
+from __future__ import absolute_import, division, print_function
+import datasets
+
+# TODO: Add BibTeX citation
+# Find for instance the citation on arxiv or on the dataset repo/website
+_CITATION = """\
+@InProceedings{hindencorp05:lrec:2014,
+  author = {Ond{\v{r}}ej Bojar and Vojt{\v{e}}ch Diatka
+            and Pavel Rychl{\'{y}} and Pavel Stra{\v{n}}{\'{a}}k
+            and V{\'{\i}}t Suchomel and Ale{\v{s}} Tamchyna and Daniel Zeman},
+  title = "{HindEnCorp - Hindi-English and Hindi-only Corpus for Machine
+            Translation}",
+  booktitle = {Proceedings of the Ninth International Conference on Language
+               Resources and Evaluation (LREC'14)},
+  year = {2014},
+  month = {may},
+  date = {26-31},
+  address = {Reykjavik, Iceland},
+  editor = {Nicoletta Calzolari (Conference Chair) and Khalid Choukri and
+     Thierry Declerck and Hrafn Loftsson and Bente Maegaard and Joseph Mariani
+     and Asuncion Moreno and Jan Odijk and Stelios Piperidis},
+  publisher = {European Language Resources Association (ELRA)},
+  isbn = {978-2-9517408-8-4},
+  language = {english}
+}
+"""
+
+# TODO: Add description of the dataset here
+# You can copy an official description
+_DESCRIPTION = """\
+This new dataset is designed to solve research in monolingual and multilingual translation task and is crafted with a lot of care."""
+
+# TODO: Add a link to an official homepage for the dataset here
+_HOMEPAGE = "https://lindat.mff.cuni.cz/repository/xmlui/handle/11858/00-097C-0000-0023-625F-0"
+
+# TODO: Add the licence for the dataset here if you can find it
+_LICENSE = "CC BY-NC-SA 3.0"
+
+# TODO: Add link to the official dataset URLs here
+# The HuggingFace dataset library don't host the datasets but only point to the original files
+# This can be an arbitrary nested dict/list of URLs (see below in `_split_generators` method)
+_URLs = "https://lindat.mff.cuni.cz/repository/xmlui/bitstream/handle/11858/00-097C-0000-0023-625F-0/hindencorp05.plaintext.gz?sequence=3&isAllowed=y"
+
+# TODO: Name of the dataset usually match the script name with CamelCase instead of snake_case
+class HindEnCorp(datasets.GeneratorBasedBuilder):
```

Comment:
HindEnCorp -> HindEncorp

this must be the camel case version of the dataset script name `hind_encorp`

Reply:
done